net-mgmt/collectd/src/opnsense/service/templates/OPNsense/collectd/collectd.conf

```diff
@@ -0,0 +1,208 @@
+{% if helpers.exists('OPNsense.collectd.general.enabled') and OPNsense.collectd.general.enabled == '1' %}
+
+{% if helpers.exists('OPNsense.collectd.general.hostname') and OPNsense.collectd.general.hostname != '' %}
+Hostname    "{{ OPNsense.collectd.general.hostname }}"
+{% endif %}
+{% if helpers.exists('OPNsense.collectd.general.fqdnlookup') and OPNsense.collectd.general.fqdnlookup == '1' %}
+FQDNLookup    true
+{% endif %}
+{% if helpers.exists('OPNsense.collectd.general.interval') and OPNsense.collectd.general.interval != '' %}
+Interval    {{ OPNsense.collectd.general.interval }}
+{% endif %}
+
+
+
+
+LoadPlugin syslog
+<Plugin syslog>
+       LogLevel info
```

Comment:
If we must really write into system log, it should be error level at most. otherwise it will flush out all useful info from the GUI page...